cpp/src/platform/windows/LogImpl.cpp

```diff
@@ -41,7 +41,7 @@ errno_t fopen_s(FILE** pFile, const char *filename, const char *mode)
 {
     if (!pFile)
     {
-        _set_errno(EINVAL);
```

Comment:
I realized that _set_errno doesn't work only with classic mingw32, it works with mingw w64. I've updated code.
By the way, why do you use custom MINGW define instead of built in __MINGW32__ and __MINGW64__ ?